README.md

```diff
@@ -16,5 +28,11 @@ This setup comes with the follwing features:
 - ES6 Support via [babeljs](https://babeljs.io/)
 - An EditorConfig setup
 
-## License / Usage
+## License
 Feel free to fork this repo and modify it according to your needs. Leave me a note if you like this setup.
+
+## Code of Conduct
+This repository contains the Open Source [Code of Conduct](code-of-conduct.md).
```

Comment:
The file is now in the master branch: https://github.com/voodoods/webpack-boilerplate/blob/master/CODE_OF_CONDUCT.md

Please adjust the filename accordingly, then we're set 👍